health/health.go

```diff
@@ -43,16 +43,19 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, _ *http.Request) {
 	writeHealthy(w)
 }
 
-func writeHeaders(status string, w http.ResponseWriter) {
-	w.Header().Set("Content-Length", strconv.Itoa(len(status)))
+func writeHeaders(statusLen int, w http.ResponseWriter) {
+	w.Header().Set("Content-Length", strconv.Itoa(statusLen))
 	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
 	w.Header().Set("X-Content-Type-Options", "nosniff")
 }
 
 func writeUnhealthy(w http.ResponseWriter) {
-	const status = "unhealthy"
+	const (
+		status    = "unhealthy"
+		statusLen = 9
```

Comment:
Since it's a constant, might as well make it a string (or do the ItoA here if you prefer).

Reply:
That's fair, I should have just made it a string in the beginning.